app/views/sections/items/ItemPackagingProductTypeView.scala.html

```diff
@@ -0,0 +1,64 @@
+@*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import models.requests.DataRequest
+@import models.sections.items.PackagingProductType
+@import models.response.referenceData.ItemPackaging
+@import viewmodels.LegendSize
+
+@this(
+    layout: templates.Layout,
+    formHelper: FormWithCSRF,
+    govukErrorSummary: GovukErrorSummary,
+    govukRadios: GovukRadios,
+    govukButton: GovukButton,
+    h1: components.h1,
+    h2: components.h2,
+    p: components.p,
+    continueOrExit: components.continueOrExit
+)
+
+@(form: Form[_], packageType: ItemPackaging, onSubmitAction: Call)(implicit request: DataRequest[_], messages: Messages)
+
+@layout(pageTitle = title(form, messages("itemPackagingProductType.title")), maybeShowActiveTrader = maybeShowActiveTrader(request)) {
+
+        @formHelper(action = onSubmitAction, 'autoComplete -> "off") {
+
+        @if(form.errors.nonEmpty) {
+            @govukErrorSummary(ErrorSummaryViewModel(form, errorLinkOverrides = Map("value" -> "value_0")))
+        }
+
+        @h2(messages("items.subHeading"), "govuk-caption-xl", hiddenContent = Some(messages("subHeading.hidden")))
+
+        @h1(messages("itemPackagingProductType.heading"))
+
+        @p(){@messages("itemPackagingProductType.p", packageType.displayName)}
```

Comment:
I think this will yield, for example `Bag (BG)`, whereas we only want the description